setup.py

```diff
@@ -119,6 +119,8 @@ def read(*names, **kwargs):
         "toml",
         # For returning exceptions from multiprocessing.Pool.map()
         "tblib",
+        # For handling progress bars
+        "tqdm==4.62.3",
```

Comment:
Do we need to be so precise in the version?

```suggestion
        "tqdm>=4.62.3",
```

Reply:
Agree. No pin or loose pin, depending on how stable the package's API is.

Reply:
Thanks for the quick reply! The precise specification of version 4.62.3 comes from [requirements.txt](https://github.com/sqlfluff/sqlfluff/blob/main/requirements.txt#L19) (and [docs/requirements.txt](https://github.com/sqlfluff/sqlfluff/blob/main/docs/requirements.txt#L5)). I'm afraid I don't know why the requirement was so specific, but I figured I should match what's in `requirements.txt` for this PR.

Reply:
I think that’s an oversight when we reviewed that. Would you mind removing the version from that too as part of this PR? Then we’ll merge this and release 0.8.1.

We generally try to avoid specific versions unless necessary to allow SQLFluff to play nicely with other projects.

Reply:
FYI @adam-tokarski

Reply:
The version of tqdm is no longer pinned in any of those files.

Reply:
There was no particular reason to pin that specific version except my habits. I should rather note that most of other packages also do not use packages versions.

Thanks for taking care of it.